bundles/org.eclipse.orion.client.ui/web/plugins/pageLinksPlugin.js

```diff
@@ -43,15 +43,15 @@ define([
 			order: 10
 		});
 		
-		if (!util.isElectron) {
+		//if (!util.isElectron) {
```

Comment:
Since we support debugger in Electron, shell should be enabled as well.

Do we want it? Should I put it in a separate patch?